packages/misc/src/lib/misc.ts

```diff
@@ -498,30 +501,37 @@ export const genRandomPath = (): string => {
   );
 };
 
-
-export const defaultMintClaimCallback = async (params: ClaimResult): Promise<string> => {
+export const defaultMintClaimCallback: MintCallback<
+  RelayClaimProcessor
+> = async (params: ClaimResult<RelayClaimProcessor>): Promise<string> => {
   try {
-    const relayUrl = "http://127.0.0.1:8081/auth/claim";
+    const relayUrl = params.relayUrl
+      ? params.relayUrl
+      : 'http://127.0.0.1:8081/auth/claim';
     const response = await fetch(relayUrl, {
-      method: "POST",
+      method: 'POST',
       body: JSON.stringify(params),
       headers: {
-        'api-key': "67e55044-10b1-426f-9247-bb680e5fe0c8_relayer",
+        'api-key': params.relayApiKey
+          ? params.relayApiKey
+          : '67e55044-10b1-426f-9247-bb680e5fe0c8_relayer',
```

Comment:
was thinking we could use the nullish coalescing operator (??) but this is actually better, it supports falsy values including `''` and `0`